src/Hello.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Hello = function(props) {
```

Comment:
Yep! The arrow syntax, I'm not sure if its possible, I'm wondering too! I'll give it a go.